### ISSUE TITLE
May I ask, are there any plans to update the current rich text editor to a more robust version in the future?

### ISSUE THREAD

**Issue by TyroneVeblen** [microfeed/microfeed#141]
The current rich text editor is not feature-rich, and I hope there will be a more comprehensive rich text editor in the future.

**Comment by wenbinf** [comment]
Absolutely, any recommendations for open-source rich text editors (for ReactJS)?

**Comment by TyroneVeblen** [comment]
Thank you very much for your reply.

What do you think of Froala? I personally think this rich text editor is considered a very complete editor.

[Froala Editor](https://github.com/froala/wysiwyg-editor)

Their official website shows that they seem to support ReactJS

[Froala Official website](https://froala.com/wysiwyg-editor/docs/framework-plugins/react/)

(I apologize for my poor programming skills, you may need to copy and paste into the address bar to access)

PS: Maybe I'm being capricious, but I hope that in the future if a new rich text editor is updated, please don't remove the HTML source editing feature if you can.

**Comment by Aicnal** [comment]
If possible, I hope it could support Markdown. And now, I always write the docment with markdown offine and output it with HTML. After that, I copy the HTML code into the microfeed html editor.
